print-radius.c

```diff
@@ -569,6 +597,8 @@ static struct attrtype {
      { "Digest-HA1",                      NULL, 0, 0, print_attr_string },
      { "SIP-AOR",                         NULL, 0, 0, print_attr_string },
      { "Delegated-IPv6-Prefix",           NULL, 0, 0, print_attr_netmask6 },
+     { "MIP6-Feature-Vector",             NULL, 0, 0, print_attr_vector64 },
+     { "MIP6-Home-Link-Prefix",           NULL, 0, 0, print_attr_netmask6 },
```

Comment:
fun fact: freeradius and wireshark both use the same format as Framed-IPv6-Prefix and Delegated-IPv6-Prefix for this structure. This might require some "be liberal in what you accept" to make it work.

Reply:
https://www.iana.org/assignments/radius-types/radius-types.xhtml lists these types the same as well.